drivers/include/l3g4200d.h

```diff
@@ -44,13 +44,13 @@
 /**
  * @brief Default address
  *
- * The address depends on the status of SDO Pin.
- * Default address corresponds to SD0 connected to GND.
- * For more information on SerialBus Address, refer Section 5.1.1
- * I2C Operation on datasheet.
+ * The address depends on the status of SDO Pin. Default
+ * address corresponds to SDO connected to GND. For more
+ * information refer section 'I2C Operation' in the
```

Comment:
Was the section number wrong here?

Reply:
Section number was right. I noticed that there could be multiple versions of datasheet, section title helps than the section number.